pages/secure/sec_adcs_certificates.rst

```diff
@@ -0,0 +1,285 @@
+
+Defining the new template is done through the ADCS management tool "*Certification Authority*". 
+
+1. Duplicate the default ADCS WebServer template, rename it to your liking.
+2. General tab:
+  a. Set the name to something recognizable, for example "Graylog Stack Template".
+  b. The software will automatically generate the internal name, which removes all spaces: "GraylogStackTemplate".
+3. Cryptography tab:
+  a. **Provider** is the *Key Storage Provider*
+  b. **Requests can use any provider available on the subject's computer** is true
+  c. **Algorithm** is *RSA 2048*
+  d. **Request hash** is *SHA256*
+  e. **Use alternate signature hash** must be set to false.
+4. Extensions tab:
+  a. **Application policies** is set to both *server auth* as well as *client auth*.
+
+If you are going to be generating all the keypairs on your issuing CA or on another management station, then you will need to add the following as well, which will allow you to export the keypair for migration to the Graylog stack servers.
+
+5. Request handling tab:
+  a. Allow the private key to be exported is set to Yes.
+
+
+Generating the keypair and certificates - preparation
+===============
+
+The following instructions assume that you generate all the keypairs on a Windows administrative workstation, or on the issuing CA itself (meaning, you'll need that extra "*Allow the private key to be exported*" flag). You can of course generate all keys on the Graylog stack servers and then simply submit the CSR (certificate signing request) to the CA.
+
+The .INF input file for the *certreq* command would look similar to the following. Note that we are referring to the template by the internal name, which does not have whitespace!::
+
+      [Version]
+      signature="$Windows NT$"
+      [NewRequest]
+      Subject="CN=hostname5.mydomain.local"
+      HashAlgorithms=SHA256
+      Keyalgorithm=RSA
+      KeyLength=2048
+      Exportable=True
+      MachineKeySet=True
+      [RequestAttributes]
+      CertificateTemplate="GraylogStackTemplate"
+      [Extensions]
+      2.5.29.17="{text}"
+      _continue=_"dns=hostname5&" 
+      _continue=_"dns=hostname5.mydomain.local&" 
+      _continue=_"dns=graylogreceiveralias.mydomain.local&" 
+      _continue=_"dns=graylogreceiveralias&" 
+      _continue=_"ipaddress=192.168.100.105&" 
+
+If you're one of the edge-cases where you will be using an older *Internet Explorer* to talk to the IP address of the host, as opposed to the hostname or its alias, you will need to add::
+
+      _continue=_"dns=192.168.100.105&" 
+
+For some reason IExplore ignores the *ipaddress* field of the SAN (subject alternative name).
+
+The above is only one of the needed .INF files; you will need one for each keypair being generated! So adjust all relevant fields and save each .INF file separately.
```

Comment:
Please bold and italicize the sentence "you will need one for each keypair being generated!"

Reply:
```suggestion
The above is only one of the needed .INF files; **you will need one for each key pair being generated**! So adjust all relevant fields and save each .INF file separately.
```